bc_obps/registration/migrations/0057_regulatedproduct_add_isregulated_and_unit.py

```diff
@@ -0,0 +1,191 @@
+# Generated by Django 5.0.9 on 2024-11-20 22:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registration', '0056_remove_historicaltransferevent_future_designated_operator_and_more'),
+    ]
+
+    def update_regulated_product_units_and_is_regulated(apps, schema_editor):
```

Comment:
should i have a test to validate all this data being added?

Reply:
I usually add tests to verify the number of items I've added. And if there are different types of data (ie regulated vs unregulated) I count those too & make sure the correct number of those have been added.
Here's an example of tests I added for emission categories: 
https://github.com/bcgov/cas-registration/blob/develop/bc_obps/reporting/tests/models/test_emission_category.py